blog/electron-30-0.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Electron 30.0.0
+date: 2024-04-16T00:00:00.000Z
+authors:
+  - name: clavin
+    url: 'https://github.com/clavin'
+    image_url: 'https://github.com/clavin.png?size=96'
+  - name: VerteDinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: electron-30-0
+---
+
+Electron 30.0.0 has been released! It includes upgrades to Chromium `124.0.6367.49`, V8 `12.4`, and Node.js `20.11.1`.
+
+---
+
+The Electron team is excited to announce the release of Electron 30.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Twitter](https://twitter.com/electronjs) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Highlights
+
+* ASAR Integrity fuse now supported on Windows
+  * Existing apps with ASAR Integrity enabled may not work on Windows if not configured correctly.
+  * Take a look at our [ASAR Integrity tutorial](https://www.electronjs.org/docs/latest/tutorial/asar-integrity) for more information.
+* Added `WebContentsView` and `BaseWindow`, deprecating & replacing `BrowserView`
+* Added `net` module to utility process
+* Implemented support for the File System API
+
+### Stack Changes
+
+- Chromium `124.0.6367.49`
+  - New in [Chrome 124](https://developer.chrome.com/blog/new-in-chrome-124/) and in [DevTools 124](https://developer.chrome.com/blog/new-in-devtools-124/)
+  - New in [Chrome 123](https://developer.chrome.com/blog/new-in-chrome-123/) and in [DevTools 123](https://developer.chrome.com/blog/new-in-devtools-123/)
+- Node `20.11.1`
+  - [Node 20.11.1 notes](https://nodejs.org/en/blog/release/v20.11.1/)
+- V8 `12.4`
+
+Electron 29 upgrades Chromium from `122.0.6261.39` to `124.0.6367.49`, Node from `20.9.0` to `20.11.1`, and V8 from `12.2` to `12.4`.
+
+### Breaking Changes
+
+#### Behavior Changed: cross-origin iframes now use Permission Policy to access features
+
+Cross-origin iframes must now specify features available to a given `iframe` via the `allow`
+attribute in order to access them.
+
+See [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#allow) for
+more information.
+
+#### Removed: The `--disable-color-correct-rendering` switch
+
+This switch was never formally documented but it's removal is being noted here regardless. Chromium itself now has better support for color spaces so this flag should not be needed.
+
+#### Behavior Changed: `BrowserView.setAutoResize` behavior on macOS
+
+In Electron 30, BrowserView is now a wrapper around the new [WebContentsView](api/web-contents-view.md) API.
```

Comment:
I think lint is failing here - we can add the full link:
```suggestion
In Electron 30, BrowserView is now a wrapper around the new [WebContentsView](https://www.electronjs.org/docs/latest/api/web-contents) API.
```

Reply:
This is still failing due to the current SNAFU with deploying the latest docs to the website - it's a legit 404.